api/pkg/controller/sessions.go

```diff
@@ -18,6 +19,13 @@ const DEBUG = true
 // this function expects the sessionQueueMtx to be locked when it is run
 func (c *Controller) getMatchingSessionFilterIndex(ctx context.Context, filter types.SessionFilter) int {
 	for i, session := range c.sessionQueue {
+
+		if filter.Older != types.Duration(0) {
+			if time.Now().Add(-time.Duration(filter.Older)).After(session.Created) {
```

Comment:
i'm not 100% sure this is the right way round, i probably need to draw it on a piece of paper :D